### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by sainikhileshreddy**

### CODE REVIEW

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@SaiNikhileshReddy I don't think we need bug report template here as there won't be any software in this repository. What say? We can have a template for reporting errors of a person's data instead.

File: .github/ISSUE_TEMPLATE/new-member.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: New Member
+about: Description of New Member
+title: ''
+labels: ''
```
Comment (aravindvnair99) [inline .github/ISSUE_TEMPLATE/new-member.md]:
```suggestion
labels: 'Add member'
```

File: .github/ISSUE_TEMPLATE/new-member.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: New Member
```
Comment (aravindvnair99) [inline .github/ISSUE_TEMPLATE/new-member.md]:
```suggestion
name: Add member
```

**Review by sainikhileshreddy — COMMENTED** [review]
I've updated.

**Review by aravindvnair99 — APPROVED** [review]

**Pull request merged**